tests/unit/backend/layers/api/test_curation_api.py

```diff
@@ -150,17 +160,17 @@ def _test(token, is_super_curator: bool = False):
                     }
                 }
             )
-            collection_id = self.generate_unpublished_collection().collection_id
+            version_id = self.generate_unpublished_collection().version_id
```

Comment:
need to test with both the version_id and collection_id

Reply:
Should now work with both, I'll add the double tests.